packages/venia-ui/upward.yml

```diff
@@ -109,6 +109,16 @@ veniaAppShell:
             inline:
                 content-type:
                     inline: text/html
+                content-security-policy:
+                    inline: upgrade-insecure-requests
+                strict-transport-security:
+                    inline: max-age=31536000
+                x-content-type-options:
+                  inline: nosniff
+                x-frame-options:
+                  inline: SAMEORIGIN
+                x-xss-protection:
+                  inline: '1; mode=block'
```

Comment:
This may be [unnecessary](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-XSS-Protection). Does WPT expect it?

Reply:
The ones included are the five specific headers WPT looks for, dings for each missing one. The browsers that don't support CSP and need this rule we probably don't support anyway, so that's a fair point. We have the option to not include them and defend whatever score WPT decides to give us.

Reply:
As long as WPT cares, we can include them.